src/client/pkg/config/config.proto

```diff
@@ -24,6 +24,13 @@ message Config {
 // DO NOT change or remove field numbers from this proto, as if you do, v1 user
 // configs will become unparseable.
 message ConfigV1 {
```

Comment:
What's the rationale behind explicitly including a version number in the message name?